packages/actor-iterator-transform-record-intermediate-results/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@comunica/actor-iterator-transform-record-intermediate-results",
+  "version": "4.0.1",
+  "description": "A record-intermediate-results iterator-transform actor",
+  "lsd:module": true,
+  "license": "MIT",
+  "homepage": "https://comunica.dev/",
```

Comment:
Can you add the `funding` entry to all package.json's which are now present in all packages since v4?